Tabloid/client/src/components/ApplicationViews.js

```diff
@@ -3,34 +3,37 @@ import { Switch, Route, Redirect } from "react-router-dom";
 import Login from "./Login";
 import Register from "./Register";
 import Hello from "./Hello";
+import { AddReactions } from "./AddReactions";
 import CategoryList from "./CategoryList";
 import PostList from "./PostList.js";
 
 export default function ApplicationViews({ isLoggedIn }) {
-    return (
-        <main>
-            <Switch>
-                <Route path="/" exact>
-                    {isLoggedIn ? <Hello /> : <Redirect to="/login" />}
-                </Route>
+  return (
+    <main>
+      <Switch>
+        <Route path="/" exact>
+          {isLoggedIn ? <Hello /> : <Redirect to="/login" />}
+        </Route>
 
-                <Route path="/posts" exact>
-                    {isLoggedIn ? <PostList /> : <Redirect to="/login" />}
-                </Route>
+        <Route path="/posts" exact>
+          {isLoggedIn ? <PostList /> : <Redirect to="/login" />}
+        </Route>
 
-                <Route path="/login">
-                    <Login />
-                </Route>
+        <Route path="/login">
+          <Login />
+        </Route>
 
-                <Route path="/register">
-                    <Register />
-                </Route>
+        <Route path="/addReactions">
```

Comment:
Feels like you might want to talk to the team about a standard naming scheme for your routes.